src/Image.tsx

```diff
@@ -7,11 +7,11 @@ export interface IImage extends IBox<HTMLImageElement> {
   responsive?: boolean;
 }
 
-export const Image: React.FunctionComponent<IImage> = props => {
+export const Image = React.forwardRef<HTMLOrSVGElement, IImage>((props, ref) => {
```

Comment:
```suggestion
export const Image = React.forwardRef<HTMLImage, IImage>((props, ref) => {
```

Reply:
I noticed that `Image` was indeed hard-coded to `img` so we _could_ be more specific here. My question is do we gain some benefit from making the type more specific?

If at some point we removed the hardcoded `img` restriction (so users could specify their own `as=SomeClass`) then we'd have to change it again. And also... IDK I'm worried that tightening the type will lead to a situation where all the ui-kit components _except_ Image work.

Reply:
Why would we ever want to remove `img` restriction for an Image component? :thinking: 
The difference might be noticeable - different (additional) properties might be exposed.

Reply:
Moreover, we do extend `HTMLImageElement` already (see `IImage` interface)

Reply:
My question is what do we _gain_ aside from potential unintended type errors? Does making the type stricter here help in any imaginable circumstance?

Reply:
(Same question applies to all your other type suggestion changes.)

Reply:
I mean, if we plan to stick to HTMLOrSvgElement, we can basically throw the typing away :P
This kind of means anything can be rendered and each event handler will need to have type guard.

Reply:
Oooooh, do you think that's why `event: any` is _needed_ now? Do you think tightening the typescript definitions might make that go away, because then it can infer the type of `event`?

Reply:
Oooh I'm super excited about that possibility, brb gotta check

Reply:
> Oooooh, do you think that's why `event: any` is _needed_ now

Nah, tried it out and still complains.

It might be useful if you have something like this:

```ts
const onClick = useCallback<MouseEventHandler<HTMLImageElement>>((e) => {
  e.target.alt
}, []);
```